event/event.yaml

```diff
@@ -3,71 +3,55 @@ components:
     Event:
       description: |-
         The optional container for event configuration.
+        To get cp-dp data both cp_events and dp_events must be explicitly enabled. 
```

Comment:
Both cp_events.enable and dp_events.enable must be explictly set to true to get control_plane_data_plane_convergence_us metric values for convergence metrics . [ Something similar ... avoid cp-dp ]